cmd/restic/cmd_generate.go

```diff
@@ -63,22 +63,22 @@ func writeManpages(dir string) error {
 }
 
 func writeBashCompletion(file string) error {
-	Verbosef("writing bash completion file to %v\n", file)
+	Warnf("writing bash completion file to %v\n", file)
```

Comment:
I'd prefer to just hide the message if stdout is not a terminal, as we normally use stderr only to print errors.
That is, please use `if stdoutIsTerminal() {` to only call `Verbosef` if stdout is a terminal.

Reply:
You're right. I'll provide an update soon.